lib/blueprinter/formatters/date_time_formatter.rb

```diff
@@ -0,0 +1,18 @@
+module Blueprinter
+  class DateTimeFormatter
+    def extract(value, options)
+      if value.respond_to?(:strftime)
+        value = format_datetime(value, options)
+      end
+      value
```

Comment:
I think we need to alter the logic here a bit in order to prevent a breaking change.
1. If `options[:datetime_format]` is passed and `!value.respond_to?(:strftime)`, we should raise the original `BlueprinterError`.
2. Otherwise, follow through with the rest of this logic.